Library/Formula/cycript.rb

```diff
@@ -0,0 +1,93 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git", :tag => "v0.9.592", :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
```

Comment:
`cycript` requires a few files to operate: `libcycript.db`, `libcycript.jar` and `libcycript.dylib`

Normally, it places them in `#{prefix}/lib`, but `brew audit` does not like that at all (`.db` is not a library, and `.jar` files are not allowed in `lib`). I patched the code that looks for these files and the code that installs them to place them in `#{prefix}/share` instead.

It's really a brew-specific change, and I haven't discussed the proper solution with @saurik yet. Personally, I think they should be able to go in `#{prefix}/lib`.